controllers/storagecluster/external_resources.go

```diff
@@ -28,6 +28,7 @@ const (
 	cephRbdStorageClassName      = "ceph-rbd"
 	cephRgwStorageClassName      = "ceph-rgw"
 	externalCephRgwEndpointKey   = "endpoint"
+	externalCephRgwTLSSecret     = "ceph-rgw-tls-cert"
```

Comment:
small nit: change  this to `CephRgwTLSSecretKey` and consume it on `cephobjecstores.go` in L161 fSprintf("%s-%s", CephRgwTLSSecretKey, cosName )

Reply:
Done